docs/changes.rst

```diff
@@ -22,6 +22,17 @@ You should generally assume that an API is internal unless you have specific
 information to the contrary.
 
 -------------------
+3.18.1 - 2017-08-15
+-------------------
+
+This release fixes a bug introduced in the previous release. If both
+``whitelist_characters`` and ``blacklist_characters`` were passed to
+:func:`~hypothesis.strategies.characters`, then an ``InvalidArgument``
+exception would be raised. This happened regardless of what values were passed.
+
+Thanks to Zac Hatfield Dodds for reporting and fixing this.
```

Comment:
I tend to hyphenate my surname online (ie "Zac Hatfield-Dodds") to avoid ambiguity with American-style middle names.